### PR TITLE
App store: Longer description and small-thumbnail

### LINKED ISSUE #40: Add some screenshots for the app store
Linking https://github.com/nextcloud/appstore/issues/403

Please also provide a small-thumbnail, see https://nextcloudappstore.readthedocs.io/en/latest/developer.html#info-xml for more information

### PR DESCRIPTION
**Pull request by korelstar**
Currently, there is no meaningful description for the Notes app in the app store. So, here is my suggestion (which should also be in `README.md`):

> The Notes app is a distraction free notes taking app. It supports formatting using [markdown](https://en.wikipedia.org/wiki/Markdown) syntax. Notes are saved as files in your Nextcloud, so you can view and edit them with every Nextcloud client. Furthermore, a separate [RESTful API](https://github.com/nextcloud/notes/wiki/API-0.2) allows for an easy integration into third-party apps such as the [Android client](https://github.com/stefan-niedermann/nextcloud-notes). Further features include marking notes as favorites and future versions will provide categories for better orgnization.

Please spell-check @nextcloud/notes 

I'm also adding a small screenshot, which fixes #40. However, this depends on nextcloud/screenshots#30

### CODE REVIEW

**Comment by Henni** [comment]
> orgnization -> organization

Otherwise sounds good.

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 [![build state](https://travis-ci.org/nextcloud/notes.png)](https://travis-ci.org/nextcloud/notes) [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/nextcloud/notes/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/nextcloud/notes/?branch=master)
 
-The Notes app is a distraction free notes taking app. It offers a [RESTful API](https://github.com/nextcloud/notes/wiki/API-0.2) for app developers. The source code is [available on GitHub](https://github.com/nextcloud/notes).
+<!-- The following paragraph should be kept synchronized with the description in appinfo/info.xml -->
+The Notes app is a distraction free notes taking app. It supports formatting using [markdown](https://en.wikipedia.org/wiki/Markdown) syntax. Notes are saved as files in your Nextcloud, so you can view and edit them with every Nextcloud client. Furthermore, a separate [RESTful API](https://github.com/nextcloud/notes/wiki/API-0.2) allows for an easy integration into third-party apps such as the [Android client](https://github.com/stefan-niedermann/nextcloud-notes). Further features include marking notes as favorites and future versions will provide categories for better orgnization.
```
Comment (BernhardPosselt) [inline README.md]: Markdown is a name so upper case :)

Comment (korelstar) [inline README.md]: changed :-)

**Comment by korelstar** [comment]
both remarks are considered now

**Comment by BernhardPosselt** [comment]
Since @Henni basically already gave a thumbs up I'm merging this ;)

**Pull request merged**